install_files/ansible-base/roles/app/tasks/mod_pagespeed.yml

```diff
@@ -0,0 +1,61 @@
+---
+# TODO remove when mod_pagespeed is an fpf deb package
```

Comment:
Why we would make this an FPF deb package? Our general goal is to try to maintain as few deb packages as possible 😄 .

My guess: that's one way to avoid installing the package over HTTPS, which is more vulnerable to MITM than installing through `apt` and authenticating via GPG.